testdata/multiple_level_doc/reqtraq_config.json

```diff
@@ -0,0 +1,67 @@
+{
+    "repoName": "multiple_level_doc",
+    "documents": [
+        {
+            "path": "TEST-120-CST.md",
+            "prefix": "TEST",
+            "level": "CST",
+            "attributes": [
+                {
+                    "name": "External Reference"
+                }
+            ]
+        },
+        {
+            "path": "TEST-100-ORD.md",
+            "prefix": "TEST",
+            "level": "SYS",
+            "parents": [
+                {
+                    "prefix": "TEST",
+                    "level": "CST"
+                },
```

Comment:
Shouldn't this also have a srcAttribute `Component Allocation` with value `^System$` ?

Reply:
Ahh yes, in this example, either system or component requirements could link up to customer requirements